deployments/broadband_coverage.yaml

```diff
@@ -1,26 +1,83 @@
-status: Converted
+status: Draft
 registry_authors:
-- Nicolas Berrios
-- Jack Fitzsimons
-tier: 1
+  - Elena Ghazi
+  - Nicolas Berrios
+  - Jack Fitzsimons
+tier: 3
 deployment:
-  name: Broadband Coverage Estimates
+  name: Broadband Coverage
   data_curator: Microsoft
-  intended_use: Geographical Broadband Statistics
-  data_product_type: Summary statistics # TODO: Is this correct?
-  data_product_region: USA
-  description: For each of the counts queries from Microsoft Services
-    Telemetry Data, we use the Laplace Mechanism, with an epsilon of 0.1, to obtain
-    diﬀerentially private counts.
-  publication_date: '2021-01-01'
-  additional_information: https://arxiv.org/pdf/2103.14035
-  # dp_flavor: TODO
+  description: Publicly available U.S. Broadband Coverage dataset that reports broadband coverage percentages at a zip code-level.
+  intended_use: Allow researchers and policy makers to understand problems related to broadband access and develop solutions to improve broadband access while simultaneously preserving the privacy of individual households.
+  data_product_type: Dataset
+  publication_date: '2021-04-01'
+  data_product_region: Global
+  data_product_sector: Technology # Telecommunication, Public Sector/Government?
+
+  dp_flavor:
+    name: Pure DP
+    # input_metric: ''
+    # bound_on_input_distance: ''
+    # output_measure: ''
+    # bound_on_output_distance: ''
+    data_domain: A collection of user records derived from data Microsoft collects as part of their ongoing work to improve their software and services performance and security. The records are generated every time a device receives an update or connects to a Microsoft service, and contain the throughput speed estimated from the connection and the device’s location, which is determined at the zip code level location data via reverse IP.
+    unprotected_quantities: The published dataset includes 32,653 zip codes. The paper notes that this was "predefined based on average number of machines in the last years".
+
   privacy_loss:
-    privacy_unit: Aggregate statistics
-    # privacy_unit_description: TODO
+    privacy_unit: |
+      User-level
+    privacy_unit_description: Neighboring datasets are defined to be the addition or deletion of a single user in the data and all possible records of that user.
     privacy_parameters:
-      epsilon: 0.1
+      epsilon: 0.2
```

Comment:
Is it clear why this was incorrect?  Is there something released twice, or a two-step release not composed together?  Anything that can go in description field.